source/extensions/filters/http/jwt_authn/jwt_cache.cc

```diff
@@ -0,0 +1,32 @@
+#include "extensions/filters/http/jwt_authn/jwt_cache.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace JwtAuthn {
+
+JwtCache::JwtCache(int cache_size) { jwt_cache_.setMaxSize(cache_size); }
+
+::google::jwt_verify::Jwt* JwtCache::find(const std::string& token) {
+  ::google::jwt_verify::Jwt* jwt_cache_lookup{};
```

Comment:
s/jwt_cache_lookup/found_jwt/

and add 

```
if (!jwt_cache_)  {
    return nullpr;
}
```

Reply:
Done